README.md

```diff
@@ -15,11 +15,14 @@
     <a href="LICENSE">
         <img src="http://img.shields.io/badge/license-MIT-brightgreen.svg" alt="MIT License">
     </a>
-    <a href="https://github.com/vapor/queues/actions">
-        <img src="https://github.com/vapor/queues/workflows/test/badge.svg" alt="Continuous Integration">
+    <a href="https://github.com/vapor/queues/actions/workflows/test.yml">
+        <img src="https://github.com/vapor/queues/actions/workflows/test.yml/badge.svg?event=push" alt="Continuous Integration">
     </a>
     <a href="https://swift.org">
-        <img src="http://img.shields.io/badge/swift-5.2-brightgreen.svg" alt="Swift 5.2">
+        <img src="http://img.shields.io/badge/swift-5.6-brightgreen.svg" alt="Swift 5.6">
+    </a>
+    <a href="https://swift.org">
```

Comment:
the same comment as https://github.com/vapor/queues-redis-driver/pull/40#discussion_r1208108890